### DISCUSSION TITLE
Why it unable to build chromium for arm64 ?

### DISCUSSION THREAD

**Discussion by farcop** [wolfi-dev/os#23712] [Q&A]
There is only chromium for amd64 arch available.
Is it possible to build chromium for arm64 ?

**Comment by xnox** [comment]
It fails to build from source at the moment, using the same steps as on x86.
https://github.com/wolfi-dev/os/actions/runs/9914955030/job/27395001826?pr=23900#step:8:462

Any help to resolve that would be appreciated.

**Reply by farcop** [reply]
Thank you for clarification!

Probably the build steps for amd64 and arm64 should be different. 
How can I build different platforms differently in the declarative yaml wolfi-os?

**Reply by xnox** [reply]
there are multiple ways of doing it - the run steps are shell and you can do different commands based on the values of arch specific variables. see vite.yaml as an example.

But also if you can make a pr that completes the build on arm64 that would as well. See the errors in the above logs. It seems like maybe bootstrap binaries for arm64 are missing / not published by upstream? or maybe more build-dependencies need to be present?

**Reply by xnox** [reply]
https://github.com/remotion-dev/chrome-build-instructions/blob/main/V2.md might help

**Comment by farcop** [comment]
Closing this because the chromium package for arm64 now exists.
Thanks!
